src/components/Drawer/Drawer.tsx

```diff
@@ -166,16 +160,22 @@ export const Drawer = ({
    * Handle "click outside"
    * 1) onClick of the area around the drawer window, close the drawer
    */
-  function handleOnClickOutside(e: MouseEvent<HTMLElement>) {
-    if (
-      isActive &&
-      dismissible &&
-      windowRef.current &&
-      !windowRef.current.contains(e.target as HTMLElement)
-    ) {
-      handleOnClose(); /* 1 */
+  useEffect(() => {
+    function handleOnClickOutside(e: MouseEvent) {
+      if (
+        isActive &&
+        dismissible &&
+        windowRef.current &&
+        !windowRef.current.contains(e.target as HTMLElement)
+      ) {
+        handleOnClose(); /* 1 */
+      }
     }
-  }
+    document.addEventListener('click', handleOnClickOutside);
+    return () => {
+      document.removeEventListener('click', handleOnClickOutside);
+    };
+  }, [isActive, windowRef]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
The previous `handleOnClickOutside` did not work (since there's it didn't detect outside clicks with no listener on the document)